src/client.js

```diff
@@ -0,0 +1,39 @@
+import isString from 'lodash/isString';
+import { Client as CoreClient, KinveyError, isDefined } from 'kinvey-js-sdk/dist/export';
+import storage from 'local-storage-fallback';
+
+class ActiveUserStorage {
+  get(key) {
+    if (!isString(key)) {
+      throw new KinveyError('ActiveUserStorage key must be a string.');
+    }
+
+    try {
+      return JSON.parse(storage.getItem(key));
+    } catch (e) {
+      return null;
```

Comment:
@thomasconner Should there be some logging here? What kind of exceptions should we expect?